File: FingoDriver/src/main/java/com/yelloco/fingodriver/models/networking/refund/TerminalData.java
package com.yelloco.fingodriver.models.networking.refund;

public class TerminalData
{
    private String location;

    public TerminalData(){
    }

    public TerminalData(String location) {
        this.location = location;
    }

    public String getLocation() {
        return location;
    }

    public void setLocation(String location) {
        this.location = location;
    }

    @Override
    public String toString() {
        return "TerminalData{" +
                "location='" + location + '\'' +
                '}';
    }
}

File: FingoDriver/src/main/java/com/yelloco/fingodriver/models/networking/refund/TerminalData.kt
package com.yelloco.fingodriver.models.networking.refund

class TerminalData
{
    var location: String? = null

    constructor() {}
    constructor(location: String?) {
        this.location = location
    }

    override fun toString(): String {
        return "TerminalData{" +
                "location='" + location + '\'' +
                '}'
    }
}